library/silverpeas

```diff
@@ -0,0 +1,9 @@
+Maintainers: Miguel Moquillon <miguel.moquillon@silverpeas.org> (@mmoqui)
+GitRepo: https://github.com/Silverpeas/docker-silverpeas-prod.git
+
+Tags: 6.0-alpha2, latest
+GitCommit: 35af9d88f66d84e4e26ebde6f59cabedfc66f387
```

Comment:
This git commit needs to be updated to the newer one: https://github.com/Silverpeas/docker-silverpeas-prod/commit/ba0eca020edbb2325455e68205724585108327a9.